core/contextmenu.js

```diff
@@ -173,6 +173,25 @@ Blockly.ContextMenu.hide = function() {
   }
 };
 
+/**
+ * Create and return a option object that can be added to a context menu's list
+ * of options.
+ * @param {string} text The text to display to the user.
+ * @param {boolean} enabled True if the option should be enabled.  If false, the
+ *     option will be visible but grayed out.
+ * @param {Function} callback The function to be called when the option is
+ *     clicked.
+ * @return {!Object} A menu option, containing text, enabled, and a callback.
+ * @package
+ */
+Blockly.ContextMenu.createOption = function(text, enabled, callback) {
+  return {
+    text: text,
+    enabled: enabled,
+    callback: callback
+  };
```

Comment:
Personally, I find the inline object easier to read, since the keys describe the values, unlike this factory function with unlabelled arguments.  Further, the object literal promotes use of the even-smaller property initializers in ES2015 (if we ever support transpiling): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015

I understand there is a tradeoff in terms of making sure all necessary values are present (TypeScript is nice in this respect).